가게 CRUD 기능 구현

ParkKyoungMin opened the pull request:
구현사항
- 가게 CRUD
- 가게 TestCode 작성
- API 테스트 완료(현재는 url에 유저 정보를 입력하여 테스트  ex) /api/stores/1?ownerId=1 )
- SoftDelete 적용

구현예정
- 튜터님 피드백 반영
- N+1 문제 추가 해결
- 에러코드 정리 및 추가
- 카테고리, 휴무일, 임시중지 등 기능 추가

ParkKyoungMin committed:
feat(store) : 가게 CRUD 기능 구현

ParkKyoungMin committed:
chore : 불필요한 import 주석 정리

yeonic commented:
경민님 고생 많으셨습니다!! 
튜터님이 하신 피드백 중에 추가로 생각난 게 있어서 알림드립니다🔔📢

- Entity가 DTO를 의존하게 되는 문제

ParkKyoungMin committed:
feat : 영업요일 및 시간 기능, 검색 기능(정렬포함) 추가
chore : 일부 형식자 및 불필요한 부분 수정

- 테스트 코드 미구현 api 테스트만 통과
- 카테고리, 휴무일(날짜) 기능 아직 구현중 (오류로 인한 롤백)
- 페이징 어노테이션 사용 전
- 에러코드 정리 전
- 의존성 정리 중 문제가 발생해 일단은 원래 버전으로 진행했습니다.튜터님께 의논 후 수정해보겠습니다.

yeonic committed:
Merge branch 'dev' into feat/store
